### ISSUE TITLE
Customize timeout to avoid

### ISSUE THREAD

**Issue by thomasglachant** [swaggerexpert/swagger-editor-validate#621]
**Is your feature request related to a problem? Please describe.**
I often have timeout error while running this job.

**Describe the solution you'd like**
Add a parameter to the action to customize timeout duration (default 10000ms).

**Additional context**

```
Error: Error while validating in Swagger Editor
Error: TimeoutError: Waiting failed: 10000ms exceeded
Error: Process completed with exit code 1.
```

**Comment by char0n** [comment]
Hi @thomasglachant,

Thanks for the suggestion. Addressed in https://github.com/swaggerexpert/swagger-editor-validate/pull/625

**Comment by char0n** [comment]
Released as https://github.com/swaggerexpert/swagger-editor-validate/releases/tag/v1.5.0

**Comment by thomasglachant** [comment]
@char0n I just try and i still got the problem...

My configuration:
```
    - name: Validat openapi spec
      uses: swaggerexpert/swagger-editor-validate@v1
      with:
        definition-file:members.json
        default-timeout: 30000
```

I got this warning:
```
Warning: Unexpected input(s) 'default-timeout', valid inputs are ['swagger-editor-url', 'definition-file', 'ignore-error']
```

And this error: 
```
Error: Error while validating in Swagger Editor
Error: TimeoutError: Waiting failed: 10000ms exceeded
Error: Process completed with exit code 1.
```

Did i do something wrong?

**Comment by char0n** [comment]
Forgot to regenerate `v1` tag. Tag was not regenerated and default-timeout input will now be accepted.
